gson/src/main/java/com/google/gson/internal/bind/ReflectiveTypeAdapterFactory.java

```diff
@@ -63,13 +63,13 @@ static boolean excludeField(Field f, boolean serialize, Excluder excluder) {
     return !excluder.excludeClass(f.getType(), serialize) && !excluder.excludeField(f, serialize);
   }
```

Comment:
Libraries and apps which depend on Gson will break unless they all are
recompiled.

On Mon, Sep 7, 2015, 5:02 PM inder123 <notifications@github.com> wrote:

> In
> gson/src/main/java/com/google/gson/internal/bind/ReflectiveTypeAdapterFactory.java
> <https://github.com/google/gson/pull/699#discussion_r38879654>:
>
> > @@ -63,13 +63,13 @@ static boolean excludeField(Field f, boolean serialize, Excluder excluder) {
> >      return !excluder.excludeClass(f.getType(), serialize) && !excluder.excludeField(f, serialize);
> >    }
>
> I am wondering may be should do a release that breaks binary compatibility
> with this and a few additional features (such as implement List for
> JsonArray).
>
> What's the big downside of breaking binary compatibility.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/gson/pull/699/files#r38879654>.
>